src/js/components/dashboard/DashboardTable.jsx

```diff
@@ -222,15 +228,19 @@ export default class DashboardTable extends React.Component {
             loadingClass = ' loading';
         }
 
-        let lastColumns = "Delete";
+        let newHeaders = null;
```

Comment:
This should default to `tableHeaders`. It's currently running into an issue going to the dashboard page caused by this being null.